config.yml

```
@@ -28,3 +28,117 @@ paths:
+
+# Request Blocking Configuration
+blocking:
+  # Block patterns are regular expressions that match URLs to block
+  patterns:
+    # Analytics and Tracking
+    - "google-analytics\\.com"
+    - "analytics\\."
+    - "tracker\\."
+    - "tracking\\."
+    - "stats\\."
+    
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)